backend/api_calls.py

```diff
@@ -728,6 +728,64 @@ def AddPlayerToChat(request, firebase):
   results.append(firebase.put('/games/%s/chatRooms' % game, chat, True))
 
 
+def Infect(request, firebase):
+  """Infect a player(otherPlayerId) and update access to chat rooms accordingly.
+
+  Validation:
+    playerId:
+    otherPlayerId:
+    gameId:
+
+    
+  """
+  results = []
+  valid_args = ['infectionId', 'playerId', 'otherPlayerId', 'gameId']
```

Comment:
Sure. If the lifeCodes doesn't exist at all, that can be done separately. Add a TODO into the code.
Also see how GroupToGame and PlayerToGame() work